cterasdk/common/defaults.py

```diff
@@ -0,0 +1,2 @@
+def default_class(CTERAHost, name):
```

Comment:
Why not implement as a CTERAHost class method?

Reply:
Makes sense. Fixed